test/new-e2e/tests/installer/unix/upgrade_scenario_test.go

```diff
@@ -600,6 +600,37 @@ func (s *upgradeScenarioSuite) TestUpgradeConfigFailure() {
 	s.mustStopExperiment(datadogAgent)
 }
 
+func (s *upgradeScenarioSuite) TestUpgradeWithProxy() {
+	if s.Env().RemoteHost.OSFlavor == e2eos.Fedora || s.Env().RemoteHost.OSFlavor == e2eos.RedHat {
+		s.T().Skip("Fedora & RedHat can't start the Squid proxy")
+	}
+
+	s.RunInstallScript("DD_REMOTE_UPDATES=true") // No proxy during install, to avoid setting up the APT proxy
```

Comment:
Not sure I follow why APT wouldn't work here ?

Reply:
It's a bit too hard to support everything that's in ansible / the install script with env vars at the moment. We should add support when we simplify the install methods